lightly/utils/benchmarking/benchmark_module.py

```diff
@@ -13,82 +11,7 @@
 from torch import Tensor
 from torch.utils.data import DataLoader
 
-# code for kNN prediction from here:
-# https://colab.research.google.com/github/facebookresearch/moco/blob/colab-notebook/colab/moco_cifar10_demo.ipynb
-
-
-def knn_predict(
```

Comment:
Moved this to `knn.py`. Backwards compatibility is maintained by importing it in `benchmarking/__init__.py`